src/Ripple/Transaction.cpp

```diff
@@ -13,7 +13,8 @@ using namespace TW;
 using namespace TW::Ripple;
 
 const int NETWORK_PREFIX = 0x53545800;
-const int64_t MAX_ALLOWED_AMOUNT = 100000000000;
+/// Max supply in drops
+const int64_t MAX_ALLOWED_AMOUNT = 100000000000 * 1000000;
```

Comment:
why do we need `MAX_ALLOWED_AMOUNT` ?

Reply:
Can we remove this option and let users to pass any number they think is correct

Reply:
You can't send coins larger than max supply

Reply:
but this is common to any blockchain, we don't have 21M limit for bitcoin transaction? can we just remove that limit

Reply:
removed